scripts/entrypoint.sh

```diff
@@ -14,11 +14,14 @@ if [ -z "${SKIP_API}" ]; then
   mkdir -p api/v2
   mkdir -p api/v3
   echo "Generating API files:"
-  ruby ./scripts/APIv*.rb
+  ruby ./scripts/APIv1.rb
+  ruby ./scripts/APIv2.rb
+  ruby ./scripts/APIv3.rb
```

Comment:
Maybe run them in a loop instead?
```suggestion
  for script in ./scripts/APIv*.rb; do
    ruby "$script"
  done
```